reconcile/openshift_base.py

```diff
@@ -36,7 +36,8 @@ class ValidationErrorJobFailed(Exception):
 
 class StateSpec:
     def __init__(self, type, oc, cluster, namespace, resource, parent=None,
-                 resource_type_override=None, resource_names=None):
+                 resource_type_override=None, resource_names=None,
+                 privileged=False):
```

Comment:
It is about time we convert this one into a dataclass, isn't it?

Reply:
On a second thought, don't change it on this PR, let's do it at some point in the near future.